src/main/kotlin/nevam/ReleaseCommand.kt

```diff
@@ -48,19 +48,24 @@ class ReleaseCommand : CliktCommand(name = "release") {
   ).default(NexusUser.DEFAULT_PASSWORD_PROPERTY)
 
   private val appModule by lazy {
+    val pomFromCoordinates = Pom(coordinates)
+    val artifactIds = pomFromCoordinates.artifactId.split(',')
```

Comment:
Hey do these IDs need to be trimmed of whitespaces?

Reply:
The docs don't say whitespaces are supported but maybe a good idea to trim anyway